src/clj/vanilla/db/core.clj

```diff
@@ -163,9 +167,11 @@
     "SQLite database connection spec."
     {:dbtype db-type :dbname "vanilla_default"})
 
-  (initialize-database vanilla-default)                     ;; Initialize this on database structure changes, push changes to repo
+  ;; Initialize this on database structure changes, push changes to repo
+  (initialize-database vanilla-default)
```

Comment:
the data above if correct, but vanilla_default hasn't been updated to refrlect the change to "date-yl-yh"